docs/install_docker.md

```diff
@@ -0,0 +1,84 @@
+# MultilateralFund Backend Development
+
+1. Install Docker and Docker Compose.
+   <https://docs.docker.com/get-docker/>
+   <https://docs.docker.com/compose/install/>
+
+2. Copy override example and adjust as necessary:
+
+   ```shell
+   cp docker-compose.develop.yml docker-compose.override.yml
+   ```
+
+3. Copy and customize the example `.env` file with configuration variables
+
+   ```shell
+   cp .env.example .env
+   ```
+
+4. Build Docker images and start the containers:
+
+   ```shell
+   docker-compose build
+   docker-compose up -d
+   ```
+
+5. Run migrations and create initial admin user:
+
+   ```shell
+   docker-compose exec app ./manage.py migrate
```

Comment:
This step should not be needed since you are running `manage.py migrate`  in the docker entrypoint.